tests/slack_sdk/web/test_web_client_issue_971.py

```diff
@@ -16,74 +16,112 @@ def tearDown(self):
         cleanup_mock_web_api_server(self)
 
     def test_text_arg_only(self):
-        client = WebClient(base_url="http://localhost:8888", token="xoxb-api_test", team_id="T111")
+        client = WebClient(
+            base_url="http://localhost:8888", token="xoxb-api_test", team_id="T111"
+        )
         resp = client.chat_postMessage(channel="C111", text="test")
         self.assertTrue(resp["ok"])
 
     def test_blocks_with_text_arg(self):
-        client = WebClient(base_url="http://localhost:8888", token="xoxb-api_test", team_id="T111")
+        client = WebClient(
```

Comment:
`./scripts/run_unit_tests.sh` does `black` formatting for you

Reply:
Good to know! (I was simply running `python setup.py validate --test-target tests/...` and I saw it running `black`, but looking through my scrollback, it wasn't doing the `tests` dir.)